src/bittytax/conv/parsers/nexo.py

```diff
@@ -144,24 +152,55 @@ def parse_nexo(data_row: "DataRow", parser: DataParser, **_kwargs: Unpack[Parser
             sell_value=value,
             wallet=WALLET,
         )
-    elif row_dict["Type"] == "Liquidation":
-        # Repayment of loan
+    # For loans, Nexo records the received amount and asset in the output columns.
+    elif row_dict["Type"] in ("WithdrawalCredit", "Loan Withdrawal"):
+        data_row.t_record = TransactionOutRecord(
+            TrType.RECEIVE_LOAN,
+            data_row.timestamp,
+            buy_quantity=buy_quantity,
+            buy_asset=buy_asset,
+            buy_value=value,
+            wallet=WALLET,
+        )
+    # These sell orders are used for repayments, but the USD value isn't recorded in the output columns.
+    elif row_dict["Type"] == "Manual Sell Order":
+        data_row.t_record = TransactionOutRecord(
+            TrType.TRADE,
+            data_row.timestamp,
+            buy_quantity=value,
+            buy_asset="USD",
+            buy_value=value,
+            sell_quantity=buy_quantity,
+            sell_asset=buy_asset,
+            sell_value=value,
+            wallet=WALLET,
+        )
     elif row_dict["Type"] in ("Liquidation", "Repayment", "Manual Repayment"):
         data_row.t_record = TransactionOutRecord(
-            TrType.SPEND,
+            TrType.REPAY_LOAN,
             data_row.timestamp,
             sell_quantity=sell_quantity,
             sell_asset=sell_asset,
-            sell_value=value,
+            sell_value=sell_quantity,
+            wallet=WALLET,
+        )
+    # "Interest Additional" is a borrowing fee for paying back a loan early. Treat as an expense.
+    elif row_dict["Type"] == "Interest Additional":
+        data_row.t_record = TransactionOutRecord(
+            TrType.BORROWING_FEE,
+            data_row.timestamp,
+            fee_quantity=sell_quantity,
+            fee_asset=sell_asset,
+            fee_value=value,
             wallet=WALLET,
         )
     # Skip loan operations which are not disposals or are just informational
     elif row_dict["Type"] in (
-        "WithdrawalCredit",
+        "Assimilation",
```

Comment:
What's "Assimilation"?

Reply:
I couldn't find an explanation for this one. I'll add this to the file too. I figure it's some admin transaction. I only found one row and it was for 0.27